mempool/src/core_mempool/mempool.rs

```diff
@@ -161,6 +161,37 @@ impl Mempool {
         }
     }
 
+    fn log_commit_and_parked_latency(insertion_info: &InsertionInfo, bucket: &str) {
+        let parked_duration = if let Some(park_time) = insertion_info.park_time {
+            let parked_duration = insertion_info
+                .ready_time
+                .duration_since(park_time)
+                .unwrap_or(Duration::ZERO);
+            counters::core_mempool_txn_commit_latency(
+                counters::PARKED_TIME_LABEL,
+                insertion_info.submitted_by_label(),
+                bucket,
+                parked_duration,
+            );
+            parked_duration
+        } else {
+            Duration::ZERO
+        };
+
+        if let Ok(commit_duration) = SystemTime::now().duration_since(insertion_info.insertion_time)
+        {
+            let commit_minus_parked = commit_duration
+                .checked_sub(parked_duration)
+                .unwrap_or(Duration::ZERO);
```

Comment:
you can replace this with:
`
let commit_minus_parked = SystemTime::now().duration_since(insertion_info.park_time.unwrap_or(insertion_info.insertion_time))`

if I understand correctly, but looks good either way

Reply:
I do want to be somewhat faithful to "minus parked" -- in some cases a transaction can end up parked much after insertion (e.g., if the sequence number before it was GC'd after some time). But to be entirely faithful the parked times would have to be a vector.. and I admit I'm not doing that

Reply:
makes sense